fearless/Modules/Pincode/PinSetup/PinSetupViewController.swift

```diff
@@ -245,8 +245,9 @@ extension PinSetupViewController: PinSetupViewProtocol {
         }
     }
 
-    func didChangeAccessoryState(enabled: Bool) {
+    func didChangeAccessoryState(enabled: Bool, accessoryIcon: UIImage?) {
```

Comment:
Maybe pass type here instead of an icon so we could check in tests that we detected all correctly